php/studentAssignQuiz.php

```diff
@@ -0,0 +1,58 @@
+<?php
+session_start();
+require_once('connection.php');
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $subjectID = $_POST['subject'];
+    $numQuestions = $_POST['question-amount'];
+    $title = "Custom Quiz";
+    $available = date('Y-m-d H:i:s');
+
+    $conn = newConn();
+
+    $stmt = $conn->prepare("CALL GetQuizSetFromSubject(?, ?)");
+    $stmt->bindParam(1, $subjectID);
+    $stmt->bindParam(2, $numQuestions, PDO::PARAM_INT);
+    $stmt->execute();
+
+    $questions = $stmt->fetchAll(PDO::FETCH_ASSOC);
+
+    $stmt->closeCursor(); //Allows the next query to execute
+
+    $conn->beginTransaction();
+
+    //Query to create the quiz
+    $stmt = $conn->prepare("CALL StudentCreateQuiz(:quizID, :subjectID, :title, :available)");
+    $quizID = bin2hex(random_bytes(16));
+    $stmt->bindValue(":quizID", $quizID, PDO::PARAM_STR);
+    $stmt->bindValue(":subjectID", $subjectID, PDO::PARAM_STR);
+    $stmt->bindValue(":title", $title, PDO::PARAM_STR);
+    $stmt->bindValue(":available", $available, PDO::PARAM_STR);
```

Comment:
This should be null, and I believe PDO has a `PARAM_NULL` for that too. An available time of NULL just means that is always available.